test/e2e_node/privileged_test.go

```diff
@@ -151,7 +150,6 @@ func (config *PrivilegedPodTestConfig) dialFromContainer(containerIP string, con
 	var output map[string]string
 	err = json.Unmarshal([]byte(stdout), &output)
 	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Could not unmarshal curl response: %s", stdout))
-	glog.Infof("Deserialized output is %v", output)
```

Comment:
I expect that message is ignored unless the test fails, in which case it should be printed out by the assertion.  That output is actually expected, but can make debugging unrelated failures hard because it looks like an error message even though it is printed at info level.

My opinion would be that anything besides the test results themselves are just debug information that we should only assume will be seen if there is a test failure.  To the extent possible, debug information should only be printed if it is related to a failed tests.